### PR TITLE
Add rename file for git ops

### PR DESCRIPTION
**Pull request by Svenito**
Add rename file in git repo. This is needed for PR akaihola/darker#545 tests to work

### CODE REVIEW

File: src/darkgraylib/testtools/git_repo_plugin.py [hunk 1]
```diff
@@ -73,6 +73,11 @@ def add(
             self._run("commit", "-m", commit)
         return absolute_paths
 
+    def rename(self, old_name: str, new_name:str , commit: str = None):
```
Comment (akaihola) [inline src/darkgraylib/testtools/git_repo_plugin.py:76]:
```suggestion
    def rename(self, old_name: str, new_name:str, commit: str = None):
```

File: src/darkgraylib/testtools/git_repo_plugin.py [hunk 2]
```diff
@@ -73,6 +73,11 @@ def add(
...
+        self._run("mv", old_name, new_name)
```
Comment (akaihola) [inline src/darkgraylib/testtools/git_repo_plugin.py:77]:
```suggestion
        """Rename/move files and optionally commit the changes

        :param old_name: The file to rename/move
        :param new_name: New name/location for the file
        :param commit: The message for the commit, or ``None`` to skip making a commit.

        """
        self._run("mv", old_name, new_name)
```

**Review by akaihola — COMMENTED** [review]
Could you add a change log entry in the `Unreleased` / `Added` section? Thanks!

**Comment by Svenito** [comment]
> Could you add a change log entry in the `Unreleased` / `Added` section? Thanks!

Sure thing. I’ll add it to the changes I’ll make tomorrow.

**Pull request merged**